operations/flags.go

```diff
@@ -255,6 +256,11 @@ func addDbSettingsFlags(flags ...cli.Flag) []cli.Flag {
 			Usage: "Database URL(s). For a replica set, list all members separated by a comma.",
 			Value: evergreen.DefaultDatabaseUrl,
 		},
+		cli.StringFlag{
+			Name:   dbCredsFileFlagName,
+			Usage:  "specify a DB credential file location",
+			EnvVar: evergreen.MongodbAuthFile,
```

Comment:
How does setting `EnvVar` on a flag work? (I assumed it would be a flag or an environment, and I'm not sure what this means. I don't see that we use this elsewhere.)

Reply:
It's used as the default when the flag isn't specified ([docs](https://github.com/urfave/cli/blob/master/docs/v1/manual.md#values-from-the-environment))

Reply:
We use it in cedar for basically all of the flags, e.g. https://github.com/evergreen-ci/cedar/blob/21fcb86a728e54a3bad1b2cf72535f5da01ee58d/operations/flags.go#L126